src/gui/Preferences.ui

```diff
@@ -1580,6 +1580,33 @@
           </layout>
          </widget>
         </item>
+        <item>
+         <widget class="QGroupBox" name="groupBoxLocalSlicer">
+          <property name="title">
+           <string>Local Slicer</string>
+          </property>
+	  <layout class="QGridLayout" name="gridLayout_72">
+
+          <item row="0" column="0">
+            <widget class="QLabel" name="labelLocalSlicer">
+             <property name="text">
+              <string>Executable name</string>
+             </property>
+             <property name="alignment">
+              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
+             </property>
+            </widget>
+           </item>
+           <item row="0" column="2">
+            <widget class="QLineEdit" name="lineEditLocalSlicer">
```

Comment:
It's a bit inconvenient to edit the filename manually. Could you add a file dialog here?

Reply:
Done